src/include/optimizer/stats/cost.h

```diff
@@ -118,7 +126,7 @@ class Cost {
    * Note right only first column is taken into consideration.
    */
   static double OrderByCost(const std::shared_ptr<TableStats>& input_stats,
-                            const std::vector<oid_t>& columns,
+                            const std::vector<std::string>& columns,
```

Comment:
Just a reminder here: in the future, the cost model should use the expression object itself to match columns since only the base columns (tuple value expressions) have valid column names. For now it is okay.

Reply:
@hzxa21  would you mind explaining more what you mean by expression object? For base table, I am guessing expression object is just value expression (table name?) but how about for intermediate tables?

Reply:
Each node in the physical operator tree is associated with a PropertyColumns which contains a collection of expressions to specify what the output of the current node should be. This expression object is hashable and comparable so it can be used directly to match both base table and intermediate table.